### PR TITLE
fix checkstyle check fail

### PR DESCRIPTION
**Pull request by jonyangx**
Fixes ISSUE#670.

### Motivation

fix checkstyle check fail.

### Modifications

update spotbug-gradle-plugin to v5.0.3.
reformat java code.

### Documentation
No

### CODE REVIEW

File: style/checkStyle.xml [hunk 1]
```diff
@@ -74,9 +74,9 @@
     </module>
     <module name="AvoidStarImport"/>
     <module name="ImportOrder">
-      <property name="staticGroups" value="org.apache.eventmesh,org.apache,java,javax,org,com"/>
+      <property name="staticGroups" value="java,javax,org,io,net,junit,com,cn,lombok,org.apache.eventmesh"/>
       <property name="separatedStaticGroups" value="true"/>
-      <property name="groups" value="org.apache.eventmesh,org.apache,java,javax,org,com"/>
+      <property name="groups" value="java,javax,org,io,net,junit,com,lombok,org.apache.eventmesh"/>
```
Comment (ruanwenjun) [inline style/checkStyle.xml]: Is there any reason to change the Import order here? Refer to some project like Flink, import apache package first looks better.

Comment (jonyangx) [reply style/checkStyle.xml]: for java spec java/javax native libray import first looks better.

Comment (jonyangx) [reply style/checkStyle.xml]: Ok,I will change it and summit it again.

Comment (jonyangx) [reply style/checkStyle.xml]: I have changed it and submitted again.

**Review by ruanwenjun — APPROVED** [review]
LGTM, if no other concern, this pr should merge quickly. Since the current master branch loss gradle file, this may block other pr. @xwm1992 @qqeasonchen

**Review by xwm1992 — APPROVED** [review]
LGTM

**Pull request merged**